raiden-ts/src/channels/epics.ts

```diff
@@ -769,21 +767,14 @@ export const channelCloseEpic = (
         return of(channelClose.failure(error, action.meta));
       }
 
-      let balanceHash = HashZero as Hash,
-        nonce = Zero as UInt<8>,
-        additionalHash = HashZero as Hash,
-        nonClosingSignature = hexlify(SignatureZero) as Signature;
-
-      if (channel.partner.balanceProof) {
-        balanceHash = createBalanceHash(
-          channel.partner.balanceProof.transferredAmount,
-          channel.partner.balanceProof.lockedAmount,
-          channel.partner.balanceProof.locksroot,
-        );
-        nonce = channel.partner.balanceProof.nonce;
-        additionalHash = channel.partner.balanceProof.messageHash;
-        nonClosingSignature = channel.partner.balanceProof.signature;
-      }
+      const balanceHash = createBalanceHash(
+        channel.partner.balanceProof.transferredAmount,
```

Comment:
nitpick: Would it be easy to extract `balanceProof` to a variable to a variable instead of repeating `channel.partner.balanceProof` everywhere?

Reply:
I'll do it here, but they're `const`/`readonly` properties which [now] are always defined, so I don't see much advantage in extracting them. It's not like accessing a deep, unrelated object. These members could as well all be on the root of the object (which would save the deep access), but are split/nested just for deduplication/code organization